### PR TITLE
Update run-all.sh & conf for all benchmarks/languages

### PR DESCRIPTION
**Pull request by gczsjdy**
Update run-all.sh to follow the current directory structure and obey the unified style with other bash files
Add benchmarks.lst & languages.lst

### CODE REVIEW

File: conf/benchmarks.lst [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+micro.sort
+micro.terasort
+micro.wordcount
+micro.dfsioe
+
+sql.aggregation
+sql.join
+sql.scan
+
+websearch.pagerank
```
Comment (carsonwang) [inline conf/benchmarks.lst]: nutchindexing is missing here.

File: conf/languages.lst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+hadoop
+spark
```
Comment (carsonwang) [inline conf/languages.lst]: Can we rename the file to frameworks.lst which seems to be more appropriate?

Comment (gczsjdy) [inline conf/languages.lst]: Same opinion! Thought would do it 'tmr' : )

**Comment by carsonwang** [comment]
Thanks @gczsjdy !

**Pull request merged**